### PR TITLE
Reorg Custom Section

### PR DESCRIPTION
**Pull request by Andyvargtz**

### CODE REVIEW

**Comment by Andyvargtz** [comment]
This PR is a reorg of the section Customize inside Subnet tab. No changes were made to the content yet

File: sidebars.js [hunk 1]
```diff
@@ -141,17 +141,36 @@ const sidebars = {
...
+        "subnets/subnet-evm-genesis",
+        "subnets/subnet-evm-precompiles",
+        "subnets/subnet-evm-avalanchego-config",
+        "subnets/subnet-evm-upgrades",
         "subnets/hello-world-precompile-tutorial",
+      ],
+    },
+    {
+      type: "category",
+      label: "Custom VM Chain",
```
Comment (yulin-dong) [inline sidebars.js:166]:
```suggestion
      label: "Customize a VM Chain",
```

File: sidebars.js [hunk 2]
```diff
@@ -141,17 +141,36 @@ const sidebars = {
     },
     {
       type: "category",
-      label: "Customize",
+      label: "Customize your Subnet",
```
Comment (yulin-dong) [inline sidebars.js:144]:
```suggestion
      label: "Customize a Subnet",
```

File: static/_redirects [hunk 3]
```diff
@@ -370,7 +370,13 @@
...
 /subnets /learn/avalanche/subnets-overview 301
 
-#05/24/2023
+# 05/24/2023
 /apis/avalanchego/avalanchego-release-notes#v1912-view-on-github https://github.com/ava-labs/avalanchego/releases/tag/v1.9.12 301
 /apis/avalanchego/avalanchego-release-notes#v1714-view-on-github https://github.com/ava-labs/avalanchego/releases/tag/v1.7.14 301
 /apis/avalanchego/avalanchego-release-notes https://github.com/ava-labs/avalanchego/releases 301
+
+# 05/31/2023
+/subnets/customize-a-subnet.md#avalanchego-chain-configs docs/subnets/subnet-evm-avalanchego-config.md 301
```
Comment (yulin-dong) [inline static/_redirects:379]: Please manually verify/correct each of these links. @meaghanfitzgerald can help you on this.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: sidebars.js [hunk 4]
```diff
@@ -141,17 +141,36 @@ const sidebars = {
...
+        "subnets/subnet-evm-genesis",
+        "subnets/subnet-evm-precompiles",
+        "subnets/subnet-evm-avalanchego-config",
+        "subnets/subnet-evm-upgrades",
         "subnets/hello-world-precompile-tutorial",
+      ],
+    },
+    {
+      type: "category",
+      label: "Customize a VM Chain",
```
Comment (yulin-dong) [inline sidebars.js:166]: As my previous comment, this should be changed to "Build a VM Chain", as most of the docs are "How to build ...."

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: sidebars.js [hunk 5]
```diff
@@ -141,17 +141,36 @@ const sidebars = {
...
         "subnets/customize-a-subnet",
+        "subnets/cross-subnet-communication"
+      ],
+    },
+    {
+      type: "category",
+      label: "Subnet-EVM Chain",
+      collapsed: true,
+      items: [
+        "subnets/customize-a-subnet",
```
Comment (yulin-dong) [inline sidebars.js:156]: Is this how you intended for users?
Screenshot 2023-06-01 at 3 38 22 PM

Comment (Andyvargtz) [reply sidebars.js:156]: Oops!  I wrote the wrong name on the side bar 🤦‍♀️. good catch!

File: sidebars.js [hunk 6]
```diff
@@ -141,17 +141,36 @@ const sidebars = {
...
+      label: "Customize your Subnet",
       collapsed: true,
       items: [
         "subnets/customize-a-subnet",
+        "subnets/cross-subnet-communication"
+      ],
+    },
+    {
+      type: "category",
+      label: "Build A Subnet-EVM Chain",
```
Comment (yulin-dong) [inline sidebars.js:153]:
```suggestion
      label: "Build a Subnet-EVM Chain",
```
If you are not sure, please utilize https://titlecase.com/

File: sidebars.js [hunk 7]
```diff
@@ -141,17 +141,36 @@ const sidebars = {
...
+        "subnets/subnet-evm-genesis",
+        "subnets/subnet-evm-precompiles",
+        "subnets/subnet-evm-avalanchego-config",
+        "subnets/subnet-evm-upgrades",
         "subnets/hello-world-precompile-tutorial",
+      ],
+    },
+    {
+      type: "category",
+      label: "Build A VM Chain",
```
Comment (yulin-dong) [inline sidebars.js:166]:
```suggestion
      label: "Build a VM Chain",
```

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 2 above)
Comment (yulin-dong) [inline sidebars.js:144]:
```suggestion
      label: "Customize Your Subnet",
```

**Review by yulin-dong — CHANGES_REQUESTED** [review]

**Comment by meaghanfitzgerald** [comment]
These changes will be integrated during the Docs revamp project

**Pull request closed without merging**